.github/workflows/build.yaml

```diff
@@ -0,0 +1,34 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Latest is going to change in the future, best to use `ubuntu-22.04` as its a stable target.